### DISCUSSION TITLE
How to use the solvers for constraint satisfaction problems?

### DISCUSSION THREAD

**Discussion by zhouyou-gu** [cvxpy/cvxpy#2430] [Q&A] [answered]
Dear community,

I have a convex constraint satisfaction problem that I would like to use CVXPY (and the supported solvers) to solve. Straightforwardly, the problem can be constructed with the objective of minimizing 0 and with the required constraints. How can I find the feasibility indicator from the solvers when the problem is either feasible or infeasible?

Any shared hints or experiences on the above will be appreciated. Thank you very much!

**Comment by phschiele** [comment, marked as answer]
You construct and solve the feasibility problem like
```py
problem = cp.Problem(cp.Minimize(0), constraints)
problem.solve()
```

Then you check `problem.status` to see if it is `"optimal"` or `"infeasible"` (or something else).
